### ISSUE TITLE
"No entities in group" with standard_configuration: false

### ISSUE THREAD

**Issue by rousveiga** [nielsfaber/scheduler-card#394]
Hello! I set `standard_configuration` to `false` because I wanted to customize my actions. I'm trying to get all my `climate` entities in the same group:

```yaml
- type: custom:scheduler-card
  title: Horario calefacción
  standard_configuration: false
  time_step: 5
  include:
    - climate
  groups:
    - name: "Climatización"
      icon: "mdi:home-thermometer-outline"
      include:
        - climate
```

However, no entities show up in the group:

![image](https://user-images.githubusercontent.com/11377258/117790002-35ce4780-b249-11eb-9ae0-8fd823934b26.png)

Am I overlooking something?

My card version is 2.1.2.

Thanks in advance.

**Comment by nielsfaber** [comment]
The card will only show the entities that have one or more actions under them.
I think it's easier not to use `standard_configuration: false` since you will have to tell the card exactly which actions you want to see per entity (type).
Another option is to combine standard config with customization on top of it.

**Comment by rousveiga** [comment]
> The card will only show the entities that have one or more actions under them.

I see. Must've overlooked that in the documentation. Thank you!

> I think it's easier not to use `standard_configuration: false` since you will have to tell the card exactly which actions you want to see per entity (type).
> Another option is to combine standard config with customization on top of it.

Yes, that was my default approach. However, I found that, as per the documentation, my actions were replacing not only the default actions, but the last custom action was overriding the custom actions defined previously.

What I wanted to do was: add all my climate entities to the card, and keep their default actions (set HVAC mode to cool and set temperature, set HVAC mode to heat and set temperature, turn off), but then add a couple custom actions. The custom actions would set the HVAC mode to heat but with a fixed temperature.

I also explored the alternative of creating only one custom action that would set the HVAC mode to heat and allow the user to select a temperature from a selection of fixed values. This also replaced the heat option from the defaults (and unexpectedly, didn't replace the "select temperature" slider with a dropdown list of options, just replaced the name of the variable).

That was the reason I thought I would need `standard_configuration: false`.
